debian/landscape-client.docs

```diff
@@ -1,2 +1,2 @@
-README
+README.md
```

Comment:
This should not be necessary.  README exists (README.md is only a symlink to it for the sake of GH).